### ISSUE TITLE
NC21

### ISSUE THREAD

**Issue by ksaro1** [Rello/audioplayer#515]
### Actual behavior
Audio Player missing from list of Multimedia appa

### Steps to reproduce the behavior
1. On Nextcloud instance select + Apps from the user browser tab
2. Select Multimedia from the list on the LHS of the page
3. 

### Additional information (optional)
The 3.0.0 version that is already installed on my Nextcloud instance also fails to find any audio files even though the Nextcloud instance itself can find the audio files under the Files browser menu tab and it can play any audio file selected via the Files menu tab.

### Server configuration
**Operating system:**
Raspbian

**Web server:**
Apache

**Database:**
mysql

**Nextcloud/ownCloud version:** (see `/status.php`)
21.0.0 Beta 4

**Audio Player version:** (see `/settings/apps`)
3.0.0

**Are you using external storage? If yes, which one:** `/local/Seagate Backup Plus Drive/webdata/family'

### Client configuration
**Browser:**
Brave

#### Nextcloud/ownCloud log
<details>
<summary>Nextcloud/ownCloud log (`/data/[nextcloud|owncloud].log`)</summary>

```json
Insert your Nextcloud/ownCloud log here
```

</details>

#### Web server error log (optional)
<details>
<summary>Web server error log</summary>

```
Insert your web server log here
```

</details>

#### Browser log (optional)
<details>
<summary>Browser log</summary>

```
Insert your browser log here
```

</details>

**Comment by Rello** [comment]
Hello,
yes, AP is not yet NC21 compatible. had the same issue with the Analytics App already.
after it is released, I will apply the fixes also to AP

**Comment by ksaro1** [comment]
> Hello,
> yes, AP is not yet NC21 compatible. had the same issue with the Analytics App already.
> after it is released, I will apply the fixes also to AP

Thanks. I was just checking that it wasn't something I'd done.... I'll wait for the update.

**Comment by kruthoff** [comment]
The same seems true for 20.0.6 and 20.0.7. No update of audio player available. Strange, it's a minor Nextcloud upgrade...

**Comment by ChrWieg** [comment]
I have the same Problem with 20.0.7 but not with 20.0.6.?

**Comment by ChrWieg** [comment]
Hello,
is there a chance to get fixed the incompatibility problem with NC 20.0.7?
![grafik](https://user-images.githubusercontent.com/39028994/108100844-bdf7e300-7086-11eb-9840-d4f7afaed83d.png)

**Comment by Rello** [comment]
Hello,
I am using that combination. I am also using AP on NC21.
what PHP version do you have?

**Comment by ChrWieg** [comment]
Hello,
thank you for your quick response.
I'm using PHP 7.4.15 / PostgreSQL 12.5. But I'm running NC with docker (20.0.6-fpm from php:7.4-fpm-buster).

**Comment by kruthoff** [comment]
The last supported version for me is NC 20.0.5. No updates available since then.
php-7.3, postgres-11, nginx-1.14

**Comment by Rello** [comment]
regarding the php 7.4 it was supposed to be fixed by NC already. they have an issue in the version check.
https://github.com/Rello/audioplayer/blob/master/appinfo/info.xml#L39
I will push a new one

**Comment by ChrWieg** [comment]
Great job, thank you very much! Problem is fixed with AP 3.1.0 with NC 20.0.6->20.0.7.
